### ISSUE TITLE
Accessibility for Main Navigation: Ensure ARIA roles, states, and properties are valid (Severity: 5)

### ISSUE THREAD

**Issue by cjwest** [SU-SWS/decanter#396]
**Accessibility for Main Navigation**
***Ensure ARIA roles, states, and properties are valid (Severity: 5)***

[Issue]
Each <nav> element has an aria-expanded attribute, which is not valid as the <nav> does not expand or control an element that expands.

[User Impact]
The user impact is likely to be minimal, but may cause confusion for some screen reader users.

[Code Reference]
<nav class="su-main-nav" aria-label="main menu" aria-expanded="false">

**To Reproduce**
1. Go to http://decanter.stanford.edu/section-components.html#kssref-components-main-navigation

**Expected behavior**
[Expected Behavior]
aria-expanded should be removed from the <nav> elements.

[Code Example]
<nav class="su-main-nav" aria-label="main menu">

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
https://stanford.levelaccess.net/public/reporting/view_instance.php?instance_id=1058182572
